doc/building.rst

```diff
@@ -15,88 +15,151 @@ These instructions are updated to work with version |version| of the Nextcloud C
 
 You have two possibilities to clone the repo.
 
-First option is As [remote URL](https://help.github.com/en/articles/which-remote-url-should-i-use) you can choose between cloning with HTTPS URL's, which is recommended or cloning with SSH URL's.
+First option is As RemoteURL_ you can choose between cloning with HTTPS URL's, which is recommended or cloning with SSH URL's.
 
-[https://github.com/nextcloud/desktop.git](https://github.com/nextcloud/desktop.git)
+.. _RemoteURL: https://help.github.com/en/articles/which-remote-url-should-i-use
+
+https://github.com/nextcloud/desktop.git
 
 When you don't have SSH key added to your GitHub account, than use HTTPS.
 
 When you no part of the nextcloud organisation, clone with HTTPS:
 
-```
-$ git clone git@github.com:nextcloud/desktop.git
-```
+.. code-block:: bash
+
+   $ git clone git@github.com:nextcloud/desktop.git
+
 
 macOS
 -----
 
-In addition to needing XCode (along with the command line tools), developing in
-the macOS environment requires extra dependencies.  You can install these
-dependencies through MacPorts_ or Homebrew_.  These dependencies are required
-only on the build machine, because non-standard libs are deployed in the app
-bundle.
+.. note:: While it is possible to do many of the following steps using GUI
+   frontends, wherever possible the Terminal commands are listed instead, in order
+   to streamline the process.
+
+1. Install Xcode from the Mac App Store:
+
+   https://apps.apple.com/app/xcode/id497799835
+
+Then, in Terminal:
+
+2. Install Xcode command line tools:
+
+   .. code-block:: bash
+
+      % xcode-select –install
+
+3. Install Homebrew from `brew.sh`_ (which will just give you the following):
+
+.. _`brew.sh`: https://brew.sh
+
+   .. code-block:: bash
+
+      % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+4. Fix Homebrew permissions. (This is `a known issue`_, and, yes, the workaround may lead to other shell warnings.)
+
+.. _`a known issue`: https://stackoverflow.com/a/63241724
+
+   .. code-block:: bash
+
+      % sudo chown -R $(whoami):admin /usr/local/* \
+         && sudo chmod -R g+rwx /usr/local/*
```

Comment:
Are you sure that is really needed? I didn't had to do that on my vm (MacOS Catalina). I would add a comment to this that says if you get permission erros, than fix the permissions like this...

Reply:
This is apparently a known issue under some circumstances. It may no longer be applicable, but I ran into it rather more recently. If you think I should do a troubleshooting section, then that's fine.